### PR TITLE
Move predictionLower and predictionIncrease to TrainingSample so that…

### PR DESCRIPTION
**Pull request by jq**
Move predictionLower and predictionIncrease to TrainingSample so that label == true and label == false can be computed differently

@mLewisLogic @deerzq

### CODE REVIEW

File: airlearner/airlearner-strategy/src/main/scala/com/airbnb/common/ml/strategy/data/BinaryTrainingSample.scala [hunk 1]
```diff
@@ -116,4 +117,22 @@ trait BinaryTrainingSample extends BinaryScoringSample {
     // todo test with trueValue
     bound * basePivot
   }
+
+  // TODO maybe need predictionHigher?
```
Comment (deerzq) [inline airlearner/airlearner-strategy/src/main/scala/com/airbnb/common/ml/strategy/data/BinaryTrainingSample.scala]: no need to have this?

File: airlearner/airlearner-strategy/src/main/scala/com/airbnb/common/ml/strategy/data/BinaryTrainingSample.scala [hunk 2]
```diff
@@ -116,4 +117,22 @@ trait BinaryTrainingSample extends BinaryScoringSample {
...
+  def predictionLower(prediction: Double): Boolean = {
+    if (label && trueValue.isDefined) {
```
Comment (deerzq) [inline airlearner/airlearner-strategy/src/main/scala/com/airbnb/common/ml/strategy/data/BinaryTrainingSample.scala]: when will we have label = True but trueValue undefined?

**Comment by deerzq** [comment]
LGTM

**Comment by mLewisLogic** [comment]
👍

**Pull request merged**